types/nightwatch/test/testElementCommands.ts

```diff
@@ -0,0 +1,505 @@
+
+    test('demo test', function() {
+        browser
+            .getAttribute('input[type=text]', 'title' , function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser.getAttribute('input[type=text]', 'title');
+        isStringOrNull(result);
+    });
+
+    after(browser => browser.end());
+});
+
+//
+// .getCssProperty
+//
+describe('getCssProperty command demo', function() {
+    before(browser => browser.url('https://www.google.com/'));
+
+    test('demo test', function() {
+        browser
+            .getCssProperty('input[type=text]', 'background-color' , function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser.getCssProperty('input[type=text]', 'background-color');
+        isString(result);
+    });
+
+    after(browser => browser.end());
+});
+
+//
+// .getElementSize
+//
+describe('getElementSize command demo', function() {
+    test('demo test', function() {
+        browser
+            .url('https://www.ecosia.org/')
+            .getElementSize('#navbartop', function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser.url('https://www.ecosia.org/').getElementSize('#__nuxt');
+        isElementSize(result);
+    });
+});
+
+//
+// .getLocation
+//
+describe('getLocation command demo', function() {
+    test('demo test', function() {
+        browser
+            .url('https://www.ecosia.org/')
+            .getLocation('#__nuxt', function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser.url('https://www.ecosia.org/').getLocation('#__nuxt');
+        console.log(result);
+    });
+});
+
+//
+// .getTagName
+//
+describe('getTagName command demo', function() {
+    before(browser => browser.url('https://www.ecosia.org/'));
+
+    test('demo test', function() {
+        browser
+            .getTagName('#__nuxt', function(result) {
+                isNightwatchAPI(this);
+                this.assert.strictEqual(result.value, 'div');
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser.getTagName('#__nuxt');
+        isString(result);
+    });
+
+    after(browser => browser.end());
+});
+
+//
+// .getText
+//
+describe('getText command demo', function() {
+    before(browser => browser.url('https://nightwatchjs.org/'));
+
+    test('demo test', function() {
+        browser
+            .getText('#top-section', function(result) {
+                isNightwatchAPI(this);
+                this.assert.strictEqual(result.value, 'Nightwatch.js\nEnd-to-end testing, the easy way.');
```

Comment:
This tests are not executed, so this will not assert the actual type. Please replace all `this.assert` with type checks, like this:
```suggestion
                isNightwatchCallbackResult<string>(result);
```